src/test/java/models/SpanishGameTestCase.java

```diff
@@ -0,0 +1,69 @@
+package models;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+
+
+public class SpanishGameTestCase {
+
+	@Test
+	public void testSpanishGameCreation() {
+		SpanishGame g = new SpanishGame();
+		assertNotNull(g);
+	}
+
+
+	@Test
+	public void testSpanishGameBuildDeck() {
+		SpanishGame g = new SpanishGame();
+        g.buildDeck();
+        assertEquals(40, g.deck.size());
+	}
+
+
+	@Test
+    public void testSpanishGameInit(){
+        SpanishGame g = new SpanishGame();
+        g.buildDeck();
+        g.shuffle();
+        assertNotEquals(2,g.deck.get(0).getValue());
```

Comment:
I've found that this test case can either pass or fail depending on the outcome of shuffle. I think we should instead produce two decks, one shuffled and one not shuffled and make sure they are not equal.